### ISSUE TITLE
Gitlab connector

### ISSUE THREAD

**Issue by lexoyo** [silexlabs/unifile#122]
Since we have a Github connector, I would love a Gitlab connector
Note that this means that you will maintain it in time, it is not a one time development

**Comment by JbIPS** [comment]
Good idea, I thought about that today since I was considering moving to GitLab and saw GitLab Pages in the feature list. I'll have to have a look at their API

**Comment by JbIPS** [comment]
[This](https://docs.gitlab.com/ce/api/commits.html#create-a-commit-with-multiple-files-and-actions) filled me with joy 😄

**Comment by lexoyo** [comment]
Is it better than github's ?

**Comment by JbIPS** [comment]
It's way easier! With GitHub's I need five calls to do that
